python/cudf/dataframe/multiindex.py

```diff
@@ -200,9 +246,16 @@ def _get_column_major(self, df, row_tuple):
         return result
 
     def __len__(self):
+        if self._codes is None:
+            if self._df is not None:
+                self._compute_levels_and_codes()
+            else:
+                return 0
         return len(self.codes[self.codes.columns[0]])
```

Comment:
Instead of the len of the codes, we should be able to do the length of the columns, no?

Reply:
I had the same thought, hadn't implemented yet.